diff --git b/helm/cilium/templates/cilium-operator/_helpers.tpl a/helm/cilium/templates/cilium-operator/_helpers.tpl
index 67ab34c..eb7c6b6 100644
--- b/helm/cilium/templates/cilium-operator/_helpers.tpl
+++ a/helm/cilium/templates/cilium-operator/_helpers.tpl
@@ -26,12 +26,21 @@
 Return cilium operator image
 */}}
 {{- define "cilium.operator.image" -}}
-{{- if .Values.operator.image.override -}}
-{{- printf "%s" .Values.operator.image.override -}}
-{{- else -}}
+{{- if not (kindIs "slice" .) }}
+{{- (fail (printf "required list, but got %q" (kindOf .))) }}
+{{- end }}
+{{- if (ne (len .) 2) }}
+{{- (fail (printf "required list of 2 arguments, but got %d" (len .))) }}
+{{- end }}
+{{- $ := index . 0 }}
+{{- with index . 1 }}
 {{- $cloud := include "cilium.operator.cloud" . }}
 {{- $imageDigest := include "cilium.operator.imageDigestName" . }}
 {{- $tag := .Values.operator.image.tag | default "" | eq "" | ternary "" (printf ":%s" .Values.operator.image.tag) }}
-{{- printf "%s-%s%s%s%s" .Values.operator.image.repository $cloud .Values.operator.image.suffix $tag $imageDigest -}}
+{{- if .Values.operator.image.override -}}
+{{- printf "%s" .Values.operator.image.override -}}
+{{- else -}}
+{{- printf "%s/%s-%s%s%s%s" $.Values.image.registry .Values.operator.image.repository $cloud .Values.operator.image.suffix $tag $imageDigest -}}
+{{- end -}}
 {{- end -}}
 {{- end -}}
