diff --git a/cilium.orig/install/kubernetes/cilium/templates/_helpers.tpl b/cilium/install/kubernetes/cilium/templates/_helpers.tpl
index 3e5429e..b96b3f8 100644
--- a/cilium.orig/install/kubernetes/cilium/templates/_helpers.tpl
+++ b/cilium/install/kubernetes/cilium/templates/_helpers.tpl
@@ -18,11 +18,20 @@ then `include "cilium.image" .Values.image`
 will return `quay.io/cilium/cilium:v1.10.1@abcdefgh`
 */}}
 {{- define "cilium.image" -}}
+{{- if not (kindIs "slice" .) }}
+{{- (fail (printf "required list, but got %q" (kindOf .))) }}
+{{- end }}
+{{- if (ne (len .) 2) }}
+{{- (fail (printf "required list of 2 arguments, but got %d" (len .))) }}
+{{- end }}
+{{- $ := index . 0 }}
+{{- with index . 1 }}
 {{- $digest := (.useDigest | default false) | ternary (printf "@%s" .digest) "" -}}
 {{- if .override -}}
 {{- printf "%s" .override -}}
 {{- else -}}
-{{- printf "%s:%s%s" .repository .tag $digest -}}
+{{- printf "%s/%s:%s%s" $.Values.image.registry .repository .tag $digest -}}
+{{- end -}}
 {{- end -}}
 {{- end -}}
 
