diff --git a/helm/Makefile.giantswarm b/helm/Makefile.giantswarm
new file mode 100644
index 0000000..52a5651
--- /dev/null
+++ b/helm/Makefile.giantswarm
@@ -0,0 +1,12 @@
+export CILIUM_REPO:=giantswarm/cilium
+export CILIUM_OPERATOR_BASE_REPO:=giantswarm/cilium-operator
+export CLUSTERMESH_APISERVER_REPO:=giantswarm/cilium-clustermesh-apiserver
+export HUBBLE_RELAY_REPO:=giantswarm/hubble-relay
+export CERTGEN_REPO:=giantswarm/cilium-certgen
+export CILIUM_NODEINIT_REPO:=giantswarm/cilium-startup-script
+export CILIUM_ENVOY_REPO:=giantswarm/cilium-envoy
+export HUBBLE_UI_BACKEND_REPO:=giantswarm/hubble-ui-backend
+export HUBBLE_UI_FRONTEND_REPO:=giantswarm/hubble-ui
+export SPIRE_INIT_REPO:=docker.io/library/busybox
+export SPIRE_SERVER_REPO:=ghcr.io/spiffe/spire-server
+export SPIRE_AGENT_REPO:=ghcr.io/spiffe/spire-agent
