diff --git a/vendor/cilium/install/kubernetes/cilium/templates/hubble/tls-cronjob/_job-spec.tpl b/helm/cilium/templates/hubble/tls-cronjob/_job-spec.tpl
index 06fb334..b45c90c 100644
--- a/vendor/cilium/install/kubernetes/cilium/templates/hubble/tls-cronjob/_job-spec.tpl
+++ b/helm/cilium/templates/hubble/tls-cronjob/_job-spec.tpl
@@ -39,6 +39,7 @@ spec:
             - "--ca-secret-namespace={{ include "cilium.namespace" . }}"
             - "--ca-secret-name=cilium-ca"
             - "--ca-common-name=Cilium CA"
+            - "--ca-enforce-validity-throughout-leaves-duration={{ .Values.certgen.enforceCAValidityThroughoutLeavesDuration }}"
           env:
             - name: CILIUM_CERTGEN_CONFIG
               value: |
