diff --git a/vendor/cilium/install/kubernetes/Makefile b/helm/Makefile
index 84b0417..d70fdff 100644
--- a/vendor/cilium/install/kubernetes/Makefile
+++ b/helm/Makefile
@@ -3,8 +3,9 @@
 
 MAKEFILE_VALUES?=Makefile.values
 include $(MAKEFILE_VALUES)
+include Makefile.giantswarm
 
-include ../../Makefile.defs
+include Makefile.defs
 
 ifeq ($(CILIUM_VERSION),)
 export CILIUM_VERSION:=v$(VERSION)
@@ -23,8 +24,8 @@ VERSION_REGEX := '[0-9]\+\.[0-9]\+\.[0-9]\+.*'
 CILIUM_CHART_REGEX := '\([vV]ersion:\) '$(VERSION_REGEX)
 
 DOCKER_RUN := $(CONTAINER_ENGINE) container run --rm \
-	--workdir /src/install/kubernetes \
-	--volume $(CURDIR)/../..:/src \
+	--workdir /src \
+	--volume $(CURDIR)/:/src \
 	--volume $(CURDIR):$(CURDIR) \
 	--user "$(shell id -u):$(shell id -g)"
 HELM_DOCS := $(DOCKER_RUN) $(HELM_TOOLBOX_IMAGE) helm-docs
@@ -73,7 +74,7 @@ cilium/values.schema.json: cilium/values.yaml
 	$(ECHO_GEN)$@
 	$(HELM_SCHEMA_GEN) -c cilium --skip-auto-generation title,description,required,default,additionalProperties
 
-update-versions: update-chart cilium/values.yaml # Update the Helm values file to point to the current version.
+update-versions: cilium/values.yaml # Update the Helm values file to point to the current version.
 
 CRDS := $(foreach path,$(patsubst %.yaml,%,$(CRD_FILES)),$(shell basename $(path)))
 RAND_FILES := $(shell grep -R -l -e "rand" -e "shuffle" -e "genPrivateKey" -e "genCA" -e "genSignedCert" -e "UUID" -e "cilium.ca.setup" --include="*.yaml" $(ROOT_DIR)/$(RELATIVE_DIR)/cilium/templates)
