diff --git a/vendor/cilium/install/kubernetes/cilium/values.yaml.tmpl b/helm/cilium/values.yaml.tmpl
index 91ad5c2..396a86f 100644
--- a/vendor/cilium/install/kubernetes/cilium/values.yaml.tmpl
+++ b/helm/cilium/values.yaml.tmpl
@@ -245,6 +245,7 @@ configDriftDetection:
   ignoredKeys: []
 # -- Agent container image.
 image:
+  registry: gsoci.azurecr.io
   # @schema
   # type: [null, string]
   # @schema
@@ -570,7 +571,7 @@ bpf:
   # @default -- `524288`
   authMapMax: ~
   # -- Enable CT accounting for packets and bytes
-  ctAccounting: false
+  ctAccounting: true
   # @schema
   # type: [null, integer]
   # @schema
@@ -655,7 +656,7 @@ bpf:
   # @schema
   # type: [null, integer]
   # @schema
-  policyMapMax: 16384
+  policyMapMax: 65536
   # -- (float64) Configure threshold for emitting pressure metrics of policy maps.
   # @schema
   # type: [null, number]
@@ -1362,7 +1363,7 @@ socketLB:
   # -- Enable socket LB
   enabled: false
   # -- Disable socket lb for non-root ns. This is used to enable Istio routing rules.
-  # hostNamespaceOnly: false
+  hostNamespaceOnly: true
   # -- Enable terminating pod connections to deleted service backends.
   # terminatePodConnections: true
   # -- Enables tracing for socket-based load balancing.
@@ -1715,7 +1716,7 @@ hubble:
       extraIpAddresses: []
   relay:
     # -- Enable Hubble Relay (requires hubble.enabled=true)
-    enabled: false
+    enabled: true
     # -- Roll out Hubble Relay pods automatically when configmap is updated.
     rollOutPods: false
     # -- Hubble-relay container image.
@@ -1731,7 +1732,14 @@ hubble:
       useDigest: ${USE_DIGESTS}
       pullPolicy: "${PULL_POLICY}"
     # -- Specifies the resources for the hubble-relay pods
-    resources: {}
+    resources:
+      requests:
+        cpu: 100m
+        memory: 128Mi
+        ephemeral-storage: 2Gi
+      limits:
+        memory: 512Mi
+        ephemeral-storage: 2Gi
     # -- Number of replicas run for the hubble-relay deployment.
     replicas: 1
     # -- Affinity for hubble-replay
@@ -1756,11 +1764,14 @@ hubble:
     # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
     tolerations: []
     # -- Additional hubble-relay environment variables.
-    extraEnv: []
+    extraEnv:
+      - name: GOPS_CONFIG_DIR
+        value: /tmp
     # -- Annotations to be added to all top-level hubble-relay objects (resources under templates/hubble-relay)
     annotations: {}
     # -- Annotations to be added to hubble-relay pods
-    podAnnotations: {}
+    podAnnotations:
+      cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
     # -- Labels to be added to hubble-relay pods
     podLabels: {}
     # PodDisruptionBudget settings
@@ -1797,9 +1808,13 @@ hubble:
         # @schema
         maxUnavailable: 1
     # -- Additional hubble-relay volumes.
-    extraVolumes: []
+    extraVolumes:
+      - emptyDir: {}
+        name: tmp-dir
     # -- Additional hubble-relay volumeMounts.
-    extraVolumeMounts: []
+    extraVolumeMounts:
+      - name: tmp-dir
+        mountPath: /tmp
     # -- hubble-relay pod security context
     podSecurityContext:
       fsGroup: 65532
@@ -1896,7 +1911,7 @@ hubble:
     # -- Enable prometheus metrics for hubble-relay on the configured port at
     # /metrics
     prometheus:
-      enabled: false
+      enabled: true
       port: 9966
       serviceMonitor:
         # -- Enable service monitors.
@@ -1958,7 +1973,7 @@ hubble:
       level: ~
   ui:
     # -- Whether to enable the Hubble UI.
-    enabled: false
+    enabled: true
     standalone:
       # -- When true, it will allow installing the Hubble UI only, without checking dependencies.
       # It is useful if a cluster already has cilium and Hubble relay installed and you just
@@ -2008,11 +2023,16 @@ hubble:
         pullPolicy: "${PULL_POLICY}"
       # -- Hubble-ui backend security context.
       securityContext:
-        # readOnlyRootFilesystem: true
+        readOnlyRootFilesystem: true
         allowPrivilegeEscalation: false
         capabilities:
           drop:
             - ALL
+        runAsGroup: 65532
+        runAsNonRoot: true
+        runAsUser: 65532
+        seccompProfile:
+          type: RuntimeDefault
       # -- Additional hubble-ui backend environment variables.
       extraEnv: []
       # -- Additional hubble-ui backend volumes.
@@ -2020,13 +2040,14 @@ hubble:
       # -- Additional hubble-ui backend volumeMounts.
       extraVolumeMounts: []
       # -- Resource requests and limits for the 'backend' container of the 'hubble-ui' deployment.
-      resources: {}
-      #   limits:
-      #     cpu: 1000m
-      #     memory: 1024M
-      #   requests:
-      #     cpu: 100m
-      #     memory: 64Mi
+      resources:
+        limits:
+          memory: 1024M
+          ephemeral-storage: 4Gi
+        requests:
+          cpu: 100m
+          memory: 64Mi
+          ephemeral-storage: 4Gi
     frontend:
       # -- Hubble-ui frontend image.
       image:
@@ -2041,11 +2062,16 @@ hubble:
         pullPolicy: "${PULL_POLICY}"
       # -- Hubble-ui frontend security context.
       securityContext:
-        # readOnlyRootFilesystem: true
+        readOnlyRootFilesystem: true
         allowPrivilegeEscalation: false
         capabilities:
           drop:
             - ALL
+        runAsGroup: 101
+        runAsNonRoot: true
+        runAsUser: 101
+        seccompProfile:
+          type: RuntimeDefault
       # -- Additional hubble-ui frontend environment variables.
       extraEnv: []
       # -- Additional hubble-ui frontend volumes.
@@ -2053,13 +2079,14 @@ hubble:
       # -- Additional hubble-ui frontend volumeMounts.
       extraVolumeMounts: []
       # -- Resource requests and limits for the 'frontend' container of the 'hubble-ui' deployment.
-      resources: {}
-      #   limits:
-      #     cpu: 1000m
-      #     memory: 1024M
-      #   requests:
-      #     cpu: 100m
-      #     memory: 64Mi
+      resources:
+        limits:
+          memory: 1024M
+          ephemeral-storage: 4Gi
+        requests:
+          cpu: 100m
+          memory: 64Mi
+          ephemeral-storage: 4Gi
       server:
         # -- Controls server listener for ipv6
         ipv6:
@@ -2071,7 +2098,8 @@ hubble:
     # -- Additional labels to be added to 'hubble-ui' deployment object
     labels: {}
     # -- Annotations to be added to hubble-ui pods
-    podAnnotations: {}
+    podAnnotations:
+      cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
     # -- Labels to be added to hubble-ui pods
     podLabels: {}
     # PodDisruptionBudget settings
@@ -2413,7 +2441,7 @@ localRedirectPolicy: false
 
 localRedirectPolicies:
   # -- Enable local redirect policies.
-  enabled: false
+  enabled: true
 
   # -- Limit the allowed addresses in Address Matcher rule of
   # Local Redirect Policies to the given CIDRs.
@@ -2423,7 +2451,7 @@ localRedirectPolicies:
   addressMatcherCIDRs: ~
 
 # To include or exclude matched resources from cilium identity evaluation
-# labels: ""
+labels: "!.*/enforce !.*fluxcd\\.io/.* !.*kubernetes\\.io/managed-by.* !job-name"
 
 # logOptions allows you to define logging options. eg:
 # logOptions:
@@ -2610,8 +2638,8 @@ pprof:
   blockProfileRate: 0
 # -- Configure prometheus metrics on the configured port at /metrics
 prometheus:
-  metricsService: false
-  enabled: false
+  metricsService: true
+  enabled: true
   port: 9962
   serviceMonitor:
     # -- Enable service monitors.
@@ -3218,18 +3246,13 @@ operator:
   # type: [null, array]
   # @schema
   tolerations:
-    - key: "node-role.kubernetes.io/control-plane"
-      operator: Exists
-    - key: "node-role.kubernetes.io/master" #deprecated
-      operator: Exists
-    - key: "node.kubernetes.io/not-ready"
-      operator: Exists
-    - key: "node.cloudprovider.kubernetes.io/uninitialized"
-      operator: Exists
+    - operator: Exists
   # -- Additional cilium-operator container arguments.
   extraArgs: []
   # -- Additional cilium-operator environment variables.
-  extraEnv: []
+  extraEnv:
+    - name: GOPS_CONFIG_DIR
+      value: /tmp
   # -- Additional cilium-operator hostPath mounts.
   extraHostPathMounts: []
   # - name: host-mnt-data
@@ -3240,9 +3263,13 @@ operator:
   #   mountPropagation: HostToContainer
 
   # -- Additional cilium-operator volumes.
-  extraVolumes: []
+  extraVolumes:
+    - emptyDir: {}
+      name: tmp-dir
   # -- Additional cilium-operator volumeMounts.
-  extraVolumeMounts: []
+  extraVolumeMounts:
+    - name: tmp-dir
+      mountPath: /tmp
   # -- Annotations to be added to all top-level cilium-operator objects (resources under templates/cilium-operator)
   annotations: {}
   # -- HostNetwork setting
@@ -3251,6 +3278,7 @@ operator:
   hostUsers: true
   # -- Security context to be added to cilium-operator pods
   podSecurityContext:
+    runAsNonRoot: true
     seccompProfile:
       type: RuntimeDefault
   # -- Annotations to be added to cilium-operator pods
@@ -3261,7 +3289,7 @@ operator:
   podDisruptionBudget:
     # -- enable PodDisruptionBudget
     # ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
-    enabled: false
+    enabled: true
     # @schema
     # type: [null, integer, string]
     # @schema
@@ -3280,13 +3308,14 @@ operator:
     unhealthyPodEvictionPolicy: null
   # -- cilium-operator resource limits & requests
   # ref: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
-  resources: {}
-  #   limits:
-  #     cpu: 1000m
-  #     memory: 1Gi
-  #   requests:
-  #     cpu: 100m
-  #     memory: 128Mi
+  resources:
+    limits:
+      memory: 2Gi
+      ephemeral-storage: 4Gi
+    requests:
+      cpu: 100m
+      memory: 256Mi
+      ephemeral-storage: 2Gi
 
   # -- Security context to be added to cilium-operator pods
   securityContext:
@@ -3294,7 +3323,12 @@ operator:
       drop:
       - ALL
     allowPrivilegeEscalation: false
-  # runAsUser: 0
+    readOnlyRootFilesystem: true
+    runAsGroup: 65532
+    runAsNonRoot: true
+    runAsUser: 65532
+    seccompProfile:
+      type: RuntimeDefault
 
   # -- Interval for endpoint garbage collection.
   endpointGCInterval: "5m0s"
@@ -3318,7 +3352,7 @@ operator:
   # -- Enable prometheus metrics for cilium-operator on the configured port at
   # /metrics
   prometheus:
-    metricsService: false
+    metricsService: true
     enabled: true
     port: 9963
     serviceMonitor:
@@ -4290,7 +4324,7 @@ authentication:
   # -- Enable authentication processing and garbage collection.
   # Note that if disabled, policy enforcement will still block requests that require authentication.
   # But the resulting authentication requests for these requests will not be processed, therefore the requests not be allowed.
-  enabled: false
+  enabled: true
   # -- Buffer size of the channel Cilium uses to receive authentication events from the signal map.
   queueSize: 1024
   # -- Buffer size of the channel Cilium uses to receive certificate expiration events from auth handlers.
@@ -4521,3 +4555,12 @@ standaloneDnsProxy:
     digest: "${STANDALONE_DNS_PROXY_DIGEST}"
     useDigest: ${USE_DIGESTS}
     pullPolicy: "${PULL_POLICY}"
+
+
+# If true, it adds an initContainer to cilium-agent pods that cleans up any legacy kube-proxy iptables rules from the node before running cilium.
+# Only makes sense when `kubeProxyReplacement` is enabled (i.e. not set to 'disabled').
+cleanupKubeProxy: false
+
+global:
+  podSecurityStandards:
+    enforced: false
