diff --git a/vendor/cilium/install/kubernetes/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl b/helm/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl
index 611e44b..b31efbf 100644
--- a/vendor/cilium/install/kubernetes/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl
+++ b/helm/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl
@@ -14,7 +14,7 @@ spec:
           type: RuntimeDefault
       containers:
         - name: certgen
-          image: {{ include "cilium.image" .Values.certgen.image | quote }}
+          image: {{ include "cilium.image" (list $ .Values.certgen.image) | quote }}
           imagePullPolicy: {{ .Values.certgen.image.pullPolicy }}
           securityContext:
             capabilities:
