diff --git a/vendor/cilium/install/kubernetes/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl b/helm/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl
index 2fdccf4..5f5ebd9 100644
--- a/vendor/cilium/install/kubernetes/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl
+++ b/helm/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl
@@ -36,6 +36,7 @@ spec:
             - "--ca-secret-namespace={{ include "cilium.namespace" . }}"
             - "--ca-secret-name=cilium-ca"
             - "--ca-common-name=Cilium CA"
+            - "--ca-enforce-validity-throughout-leaves-duration={{ .Values.certgen.enforceCAValidityThroughoutLeavesDuration }}"
           env:
             - name: CILIUM_CERTGEN_CONFIG
               value: |
