diff --git a/vendor/cilium/install/kubernetes/cilium/values.yaml.tmpl b/helm/cilium/values.yaml.tmpl
index 1911033..0b61ef7 100644
--- a/vendor/cilium/install/kubernetes/cilium/values.yaml.tmpl
+++ b/helm/cilium/values.yaml.tmpl
@@ -187,6 +187,7 @@ name: cilium
 rollOutCiliumPods: false
 # -- Agent container image.
 image:
+  registry: gsoci.azurecr.io
   # @schema
   # type: [null, string]
   # @schema
@@ -485,7 +486,7 @@ bpf:
   # @default -- `524288`
   authMapMax: ~
   # -- Enable CT accounting for packets and bytes
-  ctAccounting: false
+  ctAccounting: true
   # @schema
   # type: [null, integer]
   # @schema
@@ -564,7 +565,7 @@ bpf:
   # @schema
   # type: [null, integer]
   # @schema
-  policyMapMax: 16384
+  policyMapMax: 65536
   # @schema
   # type: [null, number, string]
   # @schema
@@ -1107,7 +1108,7 @@ socketLB:
   # -- Enable socket LB
   enabled: false
   # -- Disable socket lb for non-root ns. This is used to enable Istio routing rules.
-  # hostNamespaceOnly: false
+  hostNamespaceOnly: true
   # -- Enable terminating pod connections to deleted service backends.
   # terminatePodConnections: true
   # -- Enables tracing for socket-based load balancing.
@@ -1278,11 +1279,11 @@ hubble:
         # Switch to false if you want to self maintain the file content.
         createConfigMap: true
         # ---- Exporters configuration in YAML format.
-        content: []
-        # - name: dns
-        #   contextOptions: []
-        #   includeFilters: []
-        #   excludeFilters: []
+        content:
+          - name: all
+            contextOptions: []
+            includeFilters: []
+            excludeFilters: []
   # -- Unix domain socket path to listen to when Hubble is enabled.
   socketPath: /var/run/cilium/hubble.sock
   # -- Enables redacting sensitive information present in Layer 7 flows.
@@ -1358,7 +1359,7 @@ hubble:
       #
       #   --set hubble.redact.enabled="true"
       #   --set hubble.redact.kafka.apiKey="true"
-      apiKey: true
+      apiKey: false
   # -- An additional address for Hubble to listen to.
   # Set this field ":4244" if you are enabling Hubble Relay, as it assumes that
   # Hubble is listening on port 4244.
@@ -1440,7 +1441,7 @@ hubble:
       extraIpAddresses: []
   relay:
     # -- Enable Hubble Relay (requires hubble.enabled=true)
-    enabled: false
+    enabled: true
     # -- Roll out Hubble Relay pods automatically when configmap is updated.
     rollOutPods: false
     # -- Hubble-relay container image.
@@ -1456,7 +1457,14 @@ hubble:
       useDigest: ${USE_DIGESTS}
       pullPolicy: "${PULL_POLICY}"
     # -- Specifies the resources for the hubble-relay pods
-    resources: {}
+    resources:
+      requests:
+        cpu: 100m
+        memory: 128Mi
+        ephemeral-storage: 2Gi
+      limits:
+        memory: 512Mi
+        ephemeral-storage: 2Gi
     # -- Number of replicas run for the hubble-relay deployment.
     replicas: 1
     # -- Affinity for hubble-replay
@@ -1481,11 +1489,14 @@ hubble:
     # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
     tolerations: []
     # -- Additional hubble-relay environment variables.
-    extraEnv: []
+    extraEnv:
+      - name: GOPS_CONFIG_DIR
+        value: /tmp
     # -- Annotations to be added to all top-level hubble-relay objects (resources under templates/hubble-relay)
     annotations: {}
     # -- Annotations to be added to hubble-relay pods
-    podAnnotations: {}
+    podAnnotations:
+      cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
     # -- Labels to be added to hubble-relay pods
     podLabels: {}
     # PodDisruptionBudget settings
@@ -1517,21 +1528,28 @@ hubble:
         # @schema
         maxUnavailable: 1
     # -- Additional hubble-relay volumes.
-    extraVolumes: []
+    extraVolumes:
+      - emptyDir: {}
+        name: tmp-dir
     # -- Additional hubble-relay volumeMounts.
-    extraVolumeMounts: []
+    extraVolumeMounts:
+      - name: tmp-dir
+        mountPath: /tmp
     # -- hubble-relay pod security context
     podSecurityContext:
       fsGroup: 65532
     # -- hubble-relay container security context
     securityContext:
-      # readOnlyRootFilesystem: true
+      allowPrivilegeEscalation: false
+      readOnlyRootFilesystem: true
       runAsNonRoot: true
       runAsUser: 65532
       runAsGroup: 65532
       capabilities:
         drop:
           - ALL
+      seccompProfile:
+        type: RuntimeDefault
     # -- hubble-relay service configuration.
     service:
       # --- The type of service used for Hubble Relay access, either ClusterIP, NodePort or LoadBalancer.
@@ -1618,7 +1636,7 @@ hubble:
     # -- Enable prometheus metrics for hubble-relay on the configured port at
     # /metrics
     prometheus:
-      enabled: false
+      enabled: true
       port: 9966
       serviceMonitor:
         # -- Enable service monitors.
@@ -1657,7 +1675,7 @@ hubble:
       port: 6062
   ui:
     # -- Whether to enable the Hubble UI.
-    enabled: false
+    enabled: true
     standalone:
       # -- When true, it will allow installing the Hubble UI only, without checking dependencies.
       # It is useful if a cluster already has cilium and Hubble relay installed and you just
@@ -1706,7 +1724,17 @@ hubble:
         useDigest: true
         pullPolicy: "${PULL_POLICY}"
       # -- Hubble-ui backend security context.
-      securityContext: {}
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+            - ALL
+        readOnlyRootFilesystem: true
+        runAsGroup: 65532
+        runAsNonRoot: true
+        runAsUser: 65532
+        seccompProfile:
+          type: RuntimeDefault
       # -- Additional hubble-ui backend environment variables.
       extraEnv: []
       # -- Additional hubble-ui backend volumes.
@@ -1720,13 +1748,14 @@ hubble:
         # -- Enable readiness probe for Hubble-ui backend (requires Hubble-ui 0.12+)
         enabled: false
       # -- Resource requests and limits for the 'backend' container of the 'hubble-ui' deployment.
-      resources: {}
-      #   limits:
-      #     cpu: 1000m
-      #     memory: 1024M
-      #   requests:
-      #     cpu: 100m
-      #     memory: 64Mi
+      resources:
+        limits:
+          memory: 1024M
+          ephemeral-storage: 4Gi
+        requests:
+          cpu: 100m
+          memory: 64Mi
+          ephemeral-storage: 4Gi
     frontend:
       # -- Hubble-ui frontend image.
       image:
@@ -1740,7 +1769,17 @@ hubble:
         useDigest: true
         pullPolicy: "${PULL_POLICY}"
       # -- Hubble-ui frontend security context.
-      securityContext: {}
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+            - ALL
+        readOnlyRootFilesystem: true
+        runAsGroup: 101
+        runAsNonRoot: true
+        runAsUser: 101
+        seccompProfile:
+          type: RuntimeDefault
       # -- Additional hubble-ui frontend environment variables.
       extraEnv: []
       # -- Additional hubble-ui frontend volumes.
@@ -1748,13 +1787,14 @@ hubble:
       # -- Additional hubble-ui frontend volumeMounts.
       extraVolumeMounts: []
       # -- Resource requests and limits for the 'frontend' container of the 'hubble-ui' deployment.
-      resources: {}
-      #   limits:
-      #     cpu: 1000m
-      #     memory: 1024M
-      #   requests:
-      #     cpu: 100m
-      #     memory: 64Mi
+      resources:
+        limits:
+          memory: 1024M
+          ephemeral-storage: 4Gi
+        requests:
+          cpu: 100m
+          memory: 64Mi
+          ephemeral-storage: 4Gi
       server:
         # -- Controls server listener for ipv6
         ipv6:
@@ -1766,7 +1806,8 @@ hubble:
     # -- Additional labels to be added to 'hubble-ui' deployment object
     labels: {}
     # -- Annotations to be added to hubble-ui pods
-    podAnnotations: {}
+    podAnnotations:
+      cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
     # -- Labels to be added to hubble-ui pods
     podLabels: {}
     # PodDisruptionBudget settings
@@ -1812,9 +1853,13 @@ hubble:
         maxUnavailable: 1
     # -- Security context to be added to Hubble UI pods
     securityContext:
+      enabled: true
+      fsGroup: 1001
       runAsUser: 1001
       runAsGroup: 1001
-      fsGroup: 1001
+      runAsNonRoot: true
+      seccompProfile:
+        type: RuntimeDefault
     # -- hubble-ui service configuration.
     service:
       # -- Annotations to be added for the Hubble UI service
@@ -2044,9 +2089,9 @@ l2NeighDiscovery:
 # -- Enable Layer 7 network policy.
 l7Proxy: true
 # -- Enable Local Redirect Policy.
-localRedirectPolicy: false
+localRedirectPolicy: true
 # To include or exclude matched resources from cilium identity evaluation
-# labels: ""
+labels: "!.*/enforce !.*fluxcd\\.io/.* !.*kubernetes\\.io/managed-by.* !job-name"
 
 # logOptions allows you to define logging options. eg:
 # logOptions:
@@ -2228,8 +2273,8 @@ pprof:
   port: 6060
 # -- Configure prometheus metrics on the configured port at /metrics
 prometheus:
-  metricsService: false
-  enabled: false
+  metricsService: true
+  enabled: true
   port: 9962
   serviceMonitor:
     # -- Enable service monitors.
@@ -2788,7 +2833,9 @@ operator:
   # -- Additional cilium-operator container arguments.
   extraArgs: []
   # -- Additional cilium-operator environment variables.
-  extraEnv: []
+  extraEnv:
+    - name: GOPS_CONFIG_DIR
+      value: /tmp
   # -- Additional cilium-operator hostPath mounts.
   extraHostPathMounts: []
   # - name: host-mnt-data
@@ -2799,15 +2846,22 @@ operator:
   #   mountPropagation: HostToContainer
 
   # -- Additional cilium-operator volumes.
-  extraVolumes: []
+  extraVolumes:
+    - emptyDir: {}
+      name: tmp-dir
   # -- Additional cilium-operator volumeMounts.
-  extraVolumeMounts: []
+  extraVolumeMounts:
+    - name: tmp-dir
+      mountPath: /tmp
   # -- Annotations to be added to all top-level cilium-operator objects (resources under templates/cilium-operator)
   annotations: {}
   # -- HostNetwork setting
   hostNetwork: true
   # -- Security context to be added to cilium-operator pods
-  podSecurityContext: {}
+  podSecurityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
   # -- Annotations to be added to cilium-operator pods
   podAnnotations: {}
   # -- Labels to be added to cilium-operator pods
@@ -2816,7 +2870,7 @@ operator:
   podDisruptionBudget:
     # -- enable PodDisruptionBudget
     # ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
-    enabled: false
+    enabled: true
     # @schema
     # type: [null, integer, string]
     # @schema
@@ -2830,17 +2884,27 @@ operator:
     maxUnavailable: 1
   # -- cilium-operator resource limits & requests
   # ref: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
-  resources: {}
-  #   limits:
-  #     cpu: 1000m
-  #     memory: 1Gi
-  #   requests:
-  #     cpu: 100m
-  #     memory: 128Mi
+  resources:
+    limits:
+      memory: 1Gi
+      ephemeral-storage: 2Gi
+    requests:
+      cpu: 100m
+      memory: 256Mi
+      ephemeral-storage: 2Gi
 
   # -- Security context to be added to cilium-operator pods
-  securityContext: {}
-  # runAsUser: 0
+  securityContext:
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+      - ALL
+    readOnlyRootFilesystem: true
+    runAsGroup: 65532
+    runAsNonRoot: true
+    runAsUser: 65532
+    seccompProfile:
+      type: RuntimeDefault
 
   # -- Interval for endpoint garbage collection.
   endpointGCInterval: "5m0s"
@@ -2860,7 +2924,7 @@ operator:
   # -- Enable prometheus metrics for cilium-operator on the configured port at
   # /metrics
   prometheus:
-    metricsService: false
+    metricsService: true
     enabled: true
     port: 9963
     serviceMonitor:
@@ -3835,3 +3899,11 @@ authentication:
 enableInternalTrafficPolicy: true
 # -- Enable LoadBalancer IP Address Management
 enableLBIPAM: true
+
+# If true, it adds an initContainer to cilium-agent pods that cleans up any legacy kube-proxy iptables rules from the node before running cilium.
+# Only makes sense when `kubeProxyReplacement` is enabled (i.e. not set to 'disabled').
+cleanupKubeProxy: false
+
+global:
+  podSecurityStandards:
+    enforced: false
